pkg/common/telemetry/config.go

```diff
@@ -2,20 +2,57 @@ package telemetry
 
 import (
 	"github.com/sirupsen/logrus"
+	"time"
 )
 
 type MetricsConfig struct {
-	FileConfig  FileConfig
-	Logger      logrus.FieldLogger
-	ServiceName string
-	Sinks       []Sink
+	FileConfig       FileConfig
+	Logger           logrus.FieldLogger
+	ServiceName      string
+	Sinks            []Sink
+	TimerGranularity time.Duration
 }
 
-type FileConfig struct {
-	Prometheus *PrometheusConfig `hcl:"Prometheus"`
+type MetricsConfigBuilder struct {
```

Comment:
What is this builder providing? It seems like the only thing it is doing is providing a default to TimerGranularity if unset. Seems like a zero value for TimerGranularity is not a valid configuration and could be used by `NewMetrics` as a sentinel to indicate that it has not been set and that the default should be used.

Reply:
For simplicity, I'll just remove `TimerGranularity` from `MetricsConfig` as mentioned on the below comment and replace it with a const. That'll eliminate the need for a Builder like this, so I'll just remove this as well.